src/common/components/clusterConfiguration/utils.ts

```diff
@@ -66,11 +62,33 @@ export const getSubnetFromMachineNetworkCidr = (machineNetworkCidr?: string) =>
   return subnet?.address;
 };
 
-export const isAdvNetworkConf = (cluster: Cluster, defaultNetworkSettings: ClusterDefaultConfig) =>
-  selectClusterNetworkCIDR(cluster) !== defaultNetworkSettings.clusterNetworkCidr ||
-  selectClusterNetworkHostPrefix(cluster) !== defaultNetworkSettings.clusterNetworkHostPrefix ||
-  selectServiceNetworkCIDR(cluster) !== defaultNetworkSettings.serviceNetworkCidr ||
-  (Boolean(cluster.networkType) && cluster.networkType !== 'OpenShiftSDN');
+const serviceNetworksEqual = (array1: ServiceNetwork[], array2: ServiceNetwork[]) =>
+  array1.length === array2.length &&
+  array1.every((network) => array2.map((elem) => elem.cidr).includes(network.cidr));
+
+const clusterNetworksEqual = (array1: ClusterNetwork[], array2: ClusterNetwork[]) =>
+  array1.length === array2.length &&
+  array1.every((clusterNetwork) =>
+    array2.find(
+      (network) =>
+        network.cidr === clusterNetwork.cidr && network.hostPrefix === clusterNetwork.hostPrefix,
+    ),
+  );
+
+export const isAdvNetworkConf = (
+  cluster: Cluster,
+  defaultNetworkValues: Partial<NetworkConfigurationValues>,
+) => {
+  return !(
+    Boolean(cluster.networkType) &&
+    cluster.networkType === 'OpenShiftSDN' &&
```

Comment:
The assumption I have now, is that if we are using OVN as the default, then it needs to be changed here as well.

Reply:
I agree that this will have to be changed if SDN is no longer default for all clusters. Maybe it will also need to take into consideration whether the cluster is single-node or multi-node. I don't want to do any changed before we get a final verdict on the defaults.